app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/SceneToProject.scala

```diff
@@ -1,11 +1,11 @@
 package com.azavea.rf.datamodel
 
 import java.util.UUID
-import java.sql.Timestamp
 
 case class SceneToProject(
   sceneId: UUID,
   projectId: UUID,
+  accepted: Boolean, /* Has a Scene been accepted from an AOI check? */
```

Comment:
I'd say `fromAoi` makes it less clear. `accepted == true` means that either a user has added that Scene manually from the UI as usual, or they've accepted a suggested Scene that was brought in via AOI checks (not yet implemented).

The comment next to it should be updated to be clearer.